src/main/java/org/fcrepo/migration/Migrator.java

```diff
@@ -18,8 +19,8 @@
  */
 public class Migrator {
 
-    public static void main(String [] args) throws IOException, XMLStreamException {
-        if (args.length == 1) {
+    public static void main(final String [] args) throws IOException, XMLStreamException {
```

Comment:
If you're using Eclipse (and likely in other IDEs) you can config a save action to add final where appropriate.